flow-typed/box-ui-elements.js

```diff
@@ -644,13 +644,9 @@ type FolderMini = {
     type: 'folder',
 };
 
-type TaskStatus =
-    | typeof TASK_NEW_INCOMPLETE
-    | typeof TASK_NEW_COMPLETED
-    | typeof TASK_NEW_APPROVED
-    | typeof TASK_NEW_REJECTED;
+type TaskStatus = 'APPROVED' | 'REJECTED' | 'COMPLETED' | 'IN_PROGRESS' | 'NOT_STARTED';
```

Comment:
Can this include the `TaskCollabStatus` as a subset instead of repeating the strings?